sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -300,5 +300,102 @@ private static void AddTelemetryFromActivityEvents(Activity activity, TelemetryI
                 BaseData = exceptionData,
             };
         }
+
+        internal static void AddEnqueuedTimeToMeasurementsAndAddLinksToProperties(Activity activity, IDictionary<string, double> measurements, ref AzMonList UnMappedTags)
+        {
+            if (activity.Links != null && activity.Links.Any())
+            {
+                if (TryGetAverageTimeInQueueForBatchAndAddLinksToProperties(activity, ref UnMappedTags, out long enqueuedTime))
```

Comment:
nit: consider reducing the characters in method name. For example, `TryGetAverageQueueTimeWithLinks`